index.html

```diff
@@ -4,180 +4,145 @@
+            <!-- END left two-thirds block-->
+            
+            <!-- right one-third block-->
+            <div class="left one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                <div class="info-block center-xs">
+                    <h2>{% t home.moneropedia %}</h2>
+                    <p class="main-resources">{% t home.moneropedia_para %}</p>
+                    <p><a href="{{site.baseurl}}/resources/moneropedia/" class="orange">{% t home.moneropedia_button %}</a></p>
                 </div>
-                <!-- end left one-third block-->
-                
-                <!-- center one-third block-->
-                <div class="center one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
-                    <div class="info-block">
-                        <h2>{% t home.user_guides %}</h2>
-                        <p class="main-resources">{% t home.user_guides_para %}</p>
-                        <p><a href="{{site.baseurl}}/resources/user-guides/" class="orange">{% t home.user_guides_button %}</a></p>
-                    </div>
+            </div>
+            <!-- END right one-third block-->
+        </div>
+    </section>
+    <!-- Full Block -->
+    <section class="container about-main full col-xs-12">
+        <div class="info-block">
+            <div class="row center-xs">
+                <div class="col"><h2>{% t home.joincommunity %}</h2></div>
+            </div>
+            <div class="row middle-xs info-block-row private">
+                <div class="col-lg-6 col-md-6 col-sm-8 col-xs-8  why-text">
+                    <h3>{% t home.contributecommunity %}</h3>
+                        <p>{% t home.contributecommunity_para %}</p>
+                        <p><a href="{{site.baseurl}}/get-started/contributing/" class="btn-link btn-fixed">{% t titles.contributing %}</a></p>
                 </div>
-                <!-- end center one-third block-->
-                
-                <!-- right one-third block-->
-                <div class="right one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
-                    <div class="info-block">
-                        <h2>{% t home.faq %}</h2>
-                        <p class="main-resources">{% t home.faq_para %}</p>
-                        <p><a href="{{site.baseurl}}/get-started/faq/" class="orange">{% t home.faq_button %}</a></p>       
-                    </div>
+                <div class="col-lg-6 col-md-6 col-sm-4 col-xs-4 center-xs">
+                    <img src="/img/learn-about-monero.png" alt="Contribute" class="main-icon">
                 </div>
-                <!-- end right one-third block-->
             </div>
-        </section>
-        
-        <!-- END 3 ONE-THIRD BLOCKS ROW -->
+            <div class="row middle-xs info-block-row">
+                <div class="col-lg-6 col-md-6 col-sm-4 col-xs-4 center-xs first-md last-sm last-xs">
+                    <img src="/img/mrl-logo.png" alt="Monero Research Lab" class="main-icon">
```

Comment:
Can we get a higher res MRL logo? With my browser maximised, it gets scaled up slightly and looks very blurry.

Reply:
Fluffypony just sent me an SVG version of the logo, i will replace the png with this new version in the next push :)